starknet_py/tests/e2e/docs/quickstart/test_creating_account.py

```diff
@@ -20,6 +21,8 @@ async def test_creating_account_client(network):
     # Creates an instance of account client which is already deployed (testnet)
 
     # old AccountClient using transaction version=0 (doesn't have __validate__ function)
+    # Warning: AccountClient is deprecated! Unless you need to use transactions version=0,
```

Comment:
```suggestion
    # Warning: AccountClient is deprecated! Unless you need to use transactions with version=0,
```

Reply:
Maybe also add a note in some better place about the new Account class only supporting transactions with version 1.